src/hermes/commands/init/oauth_github.py

```diff
@@ -0,0 +1,73 @@
+import os
+import webbrowser
+from flask import Flask, request, redirect
+from requests_oauthlib import OAuth2Session
+from threading import Timer, Thread, Event
+from waitress import serve
+
+local_port = 8333
+client_id = 'Ov23ctl0gNzr9smeVIHR'
+client_secret = 'd516303374f7e55189fe74fb2af77f31a965ad57'
```

Comment:
Is this supposed to be here?

Reply:
At least the secrets should be stored somewhere else in the end.